modules/thefuck/init.zsh

```diff
@@ -0,0 +1,24 @@
+#
+# Integrates TheFuck
+#
+# Author:
+#   Luciano Mores <leslau@gmail.com>
+#
+
+if [[ -z $commands[thefuck] ]]; then
+    echo 'thefuck is not installed, you should "pip install thefuck" first'
+    return -1
+fi
+
+# Register alias
+eval "$(thefuck --alias)"
+
+fuck-command-line() {
+    local FUCK="$(THEFUCK_REQUIRE_CONFIRMATION=0 thefuck $(fc -ln -1 | tail -n 1) 2> /dev/null)"
+    [[ -z $FUCK ]] && echo -n -e "\a" && return
+    BUFFER=$FUCK
+    zle end-of-line
+}
+zle -N fuck-command-line
+# Defined shortcut keys: [Esc] [Esc]
+bindkey "\e\e" fuck-command-line
```

Comment:
So, this is the same binding which was requested for the sudo plugin? I guess that makes sense since it does pretty much the same thing with some additional tracking.